src/main/java/core/basesyntax/service/impl/CsvFileReader.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.TransactionDto;
+import core.basesyntax.service.FileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+public class CsvFileReader implements FileReader {
+    private static final String PARSE_BY = ",";
+    private static final int COUNT_OF_PARSE = 3;
+    private static final int OPERATION = 0;
+    private static final int FRUIT = 1;
+    private static final int COUNT = 2;
+
+    @Override
+    public List<TransactionDto> readData(String filePath) {
+        List<String> listOfInformation = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new java.io.FileReader(filePath))) {
+            while (reader.ready()) {
+                listOfInformation.add(reader.readLine());
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(String.format("The problem has been encountered "
+                    + "while reading file %s.)", filePath), e);
+        }
+        listOfInformation.remove(0);
+        return convertListOfLinesToTransactionDtoList(listOfInformation);
+    }
+
+    private List<TransactionDto> convertListOfLinesToTransactionDtoList(List<String> data) {
```

Comment:
it would be really great to move this logic to the separate parser service, I would recommend to do so though it is still ok this way